lib/ansiblelint/runner.py

```diff
@@ -23,9 +23,9 @@ def __init__(
             self,
             rules: "RulesCollection",
             playbook: str,
-            tags: FrozenSet[Any],
-            skip_list: FrozenSet[Any],
-            exclude_paths: List[str],
+            tags: FrozenSet[Any] = frozenset(),
+            skip_list: Optional[FrozenSet[Any]] = frozenset(),
+            exclude_paths: List[str] = [],
```

Comment:
Lists are mutable. You shouldn't use them as method defaults.

Reply:
Using an empty list is the only way to make the parameter optional without changing the API. I do not see any problem with having a list passed. 

If you want to change exclude_paths to be something else I invite you to make PR that does this, outside of this change.